src/context/auth.js

```diff
@@ -47,6 +47,7 @@ const AuthProvider = ({ children }) => {
     const res = await login(email, password);
 
     if (!res.data.token) {
+      alert(`${res.data.email}`)
```

Comment:
Let's not use alerts. They're quite annoying for users (the site becomes unusable and it takes away focus). Let's instead create error reporting for if the login does not work correctly.